cookbooks/cdo-apps/templates/default/crontab.erb

```diff
@@ -76,10 +76,10 @@ end
 
 # cronjobs that run on all instances in all environments go here:
 
-# Restart at ~02:00am PST (~ 9am UTC) as workaround for Rails 4.2 memory leak.
+# Restart at ~02:00am and ~17:00PST (~ 1am and 9am UTC) as workaround for Rails 4.2 memory leak.
```

Comment:
nit: Pacific is UTC-8 now (the comment was correct when it was written but is not now because daylight savings time)